lib/preview-frame.ts

```diff
@@ -21,6 +25,24 @@ function loadP5(version: string, cb?: () => void) {
   document.body.appendChild(script);
 }
 
+function loadScripts(urls: string[], cb?: () => void) {
+  cb = cb || (() => {});
+
+  let i = 0;
+  let loadNextScript = () => {
+    if (i === urls.length) {
+      return cb();
+    }
+    loadScript(urls[i++], loadNextScript);
+  };
+
+  loadNextScript();
+}
+
+function p5url(version: string) {
+  return `//cdnjs.cloudflare.com/ajax/libs/p5.js/${version}/p5.js`;
```

Comment:
Yep, this is actually filed in #48 actually--I'd prefer to keep this PR nice and small, and add support for that in a future one. 😄